src/main/resources/Development-documentation/gerrit-scenarios.md

```diff
@@ -131,27 +159,84 @@ The user does the following:
 - Squashes the commits to one commits (`C1`)
 - Pushes to `refs/for/[branch name]` (`P1`)
 - Receives comments from reviewer
-- Updates code
+- Updates the code
 - Does `commit --amend` (`C2`)
-- Pushes to `refs/for/[branch name]`  (`P2`)
+- Pushes to `refs/for/[branch name]` (`P2`)
+- Gets ok from the reviewer
+- Hits the submit button in Gerrit
+- Gerrit will rebase (`C3`)
+
+Eiffel events sent from the plugin:
+
+- SCC(`E1`) sent for `P1` push with `BASE` link set to `E0`
+- SCC(`E2`) sent for `P2` push with `PREVIOUS_VERSION` link set `E1` and `BASE` link set to `E0`
+- SCS(`E3`) sent at submit with `CHANGE` link set `E2` and `PREVIOUS_VERSION` link set to `E01`
```

Comment:
Maybe it's good to mention that there isn't any event sent for the actual rebase patchset that gerrit creates in the background and that we are not linking to the actual latest "patchset" in this change link but rather the latest patchset that the user uploaded. Maybe as a "NOTE" below this.

Reply:
Fixed in 187495d